sinch/domains/verification/models/responses.py

```diff
@@ -0,0 +1,55 @@
+from dataclasses import dataclass
+from sinch.core.models.base_model import SinchBaseModel
+
+
+@dataclass
+class StartVerificationResponse(SinchBaseModel):
+    id: str
+    method: str
+    sms: dict
+    _links: list
+
+
+@dataclass
+class VerificationResponse(SinchBaseModel):
+    id: str
+    method: str
+    status: str
+    price: dict
+    identity: str
+    countryId: str
+    verificationTimestamp: str
+    reference: str
+
+
+@dataclass
+class ReportVerificationResponse(SinchBaseModel):
```

Comment:
This do not fit with specs for report response: different for sms, flashcall, ...

Reply:
TBH it fits all of the mentioned variants.
Everything here is described as a string:

<img width="712" alt="Screenshot 2024-01-18 at 16 27 42" src="https://github.com/sinch/sinch-sdk-python/assets/19619849/211d97f7-e4cc-4407-a2df-657bfd37f910">

Reply:
see above about using class for getting report report and status

Reply:
fixed